### ISSUE TITLE
Server monitoring

### ISSUE THREAD

**Issue by genkimaps** [genkimaps/gis-hub#111]
Install and configure monit service. Using the latest version of monit (5.28.0).  Installed from: https://mmonit.com/monit/dist/binary/5.28.0/monit-5.28.0-linux-x64.tar.gz using setup guide: https://mmonit.com/wiki/Monit/Installation.

The monit config file is at: `/etc/monit/monitrc`.  To edit, use `sudo nano /etc/monit/monitrc`. 

For SSL we had to change permissions on `/etc/letsencrypt/live/gis-hub.ca/privkey.pem` to 700. 

https://computingforgeeks.com/how-to-install-and-configure-monit-on-ubuntu/

Todo:  
- how do we attach monit to ckan--multiple wsgi processes are running

**Comment by genkimaps** [comment]
Email alerts are working, using this guide:  https://guides.wp-bullet.com/configure-monit-send-email-alerts-mailgun

I opened port 2812 in AWS, but can't get Monit web interface to work. This is less important, defer it for now.

**Comment by cole-fields** [comment]
Looks good, thanks for setting this up. I tried a couple things to try and get the web interface running, just out of curiosity. One approach I tried was from here: https://serverfault.com/questions/564574/monit-cant-connect-to-the-http-interface - changing the SSL settings, but that didn't seem to work. 

```
set httpd port 2812
    allow admin:<password>      # require user 'admin' with password 'monit'
    with ssl {            # enable SSL/TLS and set path to server certificate
        pemchain: /etc/letsencrypt/live/gis-hub.ca/fullchain.pem
        pemkey: /etc/letsencrypt/live/gis-hub.ca/privkey.pem
    }


```

Anyways, as you mentioned, the web interface is less important so I'm not concerned about it but am curious why it wouldn't work if the port is open and the service is listening.
